modules/statistics/AnalyticsAdapter.js

```diff
@@ -64,14 +64,15 @@ class AnalyticsAdapter {
      */
     constructor() {
         this.disposed = false;
-        this.browserName = RTCBrowserType.getBrowserName();
```

Comment:
Oh wow, I absolutely appreciate getting rid of extra instance variables (even if this is more of a move).